onfido/models/base.py

```diff
@@ -286,6 +276,11 @@ def mark_as_clear(self, user: settings.AUTH_USER_MODEL) -> Event:
 
         """
         self._override_event(user).save()
```

Comment:
I've just noticed that the `_overrride_event` fake event doesn't include the object result field. If you add this we can record the result as well:

```suggestion
    event = self._override_event(user)
    event["object"]["result"] = self.Result.CLEAR
    event.save()
```